unified_planning/test/test_trajectory_constraints_remover.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2021 AIPlan4EU project
```

Comment:
In this test, I was expecting a full usage of the `Compiler`, how it's interface should be used, how the `map_back_actions_instance` callable works and some testing on the general `CompilerResult` to make sure everything works as intended.

Try to check the tests on other compilers if you need some examples.